assets/scss/03-organisms/_key-message.scss

```diff
@@ -15,15 +15,12 @@ $colors: (
     min-height: 140px;
```

Comment:
@ygannett --> this look good - not sure if you wanted to remove this min-height in addition to the others.

Reply:
Thank you.  Good catch. Just pushed the change.